src/utils/helpers.js

```diff
@@ -70,12 +73,37 @@ export const createErrorHandler = prefix => ({ message }) => ({
 	error: `${prefix}: ${message}`,
 });
 
+const validateOutputFormatOptions = options => {
+	if (options.json && options.table) {
+		throw new Error('Cannot output both JSON and table.');
```

Comment:
In a validation function, prefer to throw the validation error

Reply:
The validation error is only in `0.3.0` so far. We can add this when `0.4.0` incorporates those changes.